db/models/__init__.py

```diff
@@ -106,6 +106,8 @@ class FlowSubDoc(BaseModel):
     """Flow DB SubDocument Model."""
 
     table_id = 0
+    owner: Optional[str]
```

Comment:
Is it safe to assume that the flow created by `flow_manager` are going to be ignored by `of_multi_table`? These are created by the user through an API request.

Reply:
@Alopalao on `v2/stored_flows?state=installed` you'll get all flows that `flow_manager` manages. `flow_manager` creates/sends flows on behalf of another NApp, do you [mean ignored cookies or tables](https://github.com/kytos-ng/flow_manager/blob/master/settings.py#L9-L10)? If you mean these ones, these won't be in the db collection. Let me know if you meant something else here.

Reply:
I mean created through API for example a flow sent to  `http://localhost:8181/api/kytos/flow_manager/v2/flows/00:00:00:00:00:00:00:01` as a `POST` request. These flows are in full control of the user. Its cookie will be `0x0` too so it is consistent with the lack of the field `owner`. I just wanted to make sure that these will not be modified by `of_multi_table`.

Reply:
Yes, having cookie 0 and None (null) owner is the expected behavior we need here when a NApp hasn't installed it, just so `of_multi_table` won't modify them. Feel free to close the thread if that clarified your concern.

Reply:
A'right. Thanks for the clarification. Closing thread.